examples/js/loaders/OBJLoader.js

```diff
@@ -40,315 +65,431 @@ THREE.OBJLoader.prototype = {
 
 	},
 
-	parse: function ( text ) {
-
-		console.time( 'OBJLoader' );
+	_createParserState : function()
```

Comment:
@jonnenauha What do you think about moving this method back inside `parse` and keep the prototype clean?